lib/plugins/aws/invokeLocal/invoke.rb

```diff
@@ -60,6 +60,8 @@ def log(message)
   handler_method, handler_class = handler_name.split(".").reverse
   handler_class ||= "Kernel"
 
+  $stdin.reopen "/dev/tty", "a+" unless Gem.win_platform? || $stdin.tty?
```

Comment:
Could this fail if there is no tty? In python invoke local it spawns the `tty` executable to make sure opening the tty will work before doing so.

Reply:
You're right - addressed in 3bfc724.